### PR TITLE
Fixes #7671: Propagate promises error when no nodes behind a relay

### PR DESCRIPTION
**Pull request by ncharles**
https://www.rudder-project.org/redmine/issues/7671

### CODE REVIEW

**Comment by ncharles** [comment]
wait - it fails on rudder server

File: techniques/system/distributePolicy/1.0/propagatePromises.st [hunk 1]
```diff
@@ -43,6 +43,14 @@ bundle agent propagatePromises
       "pass2" expression => "pass1";
       "pass1" expression => "any";
 
+    # Class definition for no propagate promises transfert, but correct promises transfert -> folder for promises for relay is empty on the server, or folder inexistent
```
Comment (amousset) [inline techniques/system/distributePolicy/1.0/propagatePromises.st]: transfer

**Comment by ncharles** [comment]
Commit modified

**Comment by amousset** [comment]
Ready for merge!

**Comment by ncharles** [comment]
ok, leaving the merge to be automated

**Comment by ncharles** [comment]
OK, merging this PR

**Comment by ncharles** [comment]
OK, merging this PR

**Pull request merged**